src/gluonts/model/forecast.py

```diff
@@ -392,7 +392,7 @@ def alpha_for_percentile(p):
 
         if show_mean:
             mean_data = np.mean(self._sorted_samples, axis=0)
-            pd.Series(data=mean_data, index=self.index).plot(
+            pd.Series(data=mean_data, index=self.index.to_timestamp()).plot(
```

Comment:
Should we have this separately?

Reply:
What I mean is: it would be nice to understand the issue with `self.index` better, so a dedicated PR may be worth it

Reply:
Probably.